fe/fe-core/src/main/java/com/starrocks/analysis/UpdateBackendAddressClause.java

```diff
@@ -0,0 +1,52 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.analysis;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.starrocks.common.AnalysisException;
+import com.starrocks.common.Pair;
+import com.starrocks.system.SystemInfoService;
+import com.google.common.base.Preconditions;
+import com.google.common.base.Strings;
+
+public class UpdateBackendAddressClause extends BackendClause {
+
+    protected Pair<String, Integer> originalHostPortPair;
+    protected String originalHostPort;
+    protected Pair<String, Integer> newhostPortPair;
+    protected String newHostPort;
```

Comment:
original and new is the non-sense name. Everyone who first read the code can not get the point.
It's better to get a concise name like IPHostpair, FQDNHostPortPair.

Reply:
If user already have modified ip to fqdn, they still can modify fqdn to another new fqdn, so the original host info not only be ip , it could be fqdn. So I think originalHostPortPair is not a non-sense name，it dose means original.